hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/utils/HddsVolumeUtil.java

```diff
@@ -143,11 +145,16 @@ public static int getLayOutVersion(Properties props, File versionFile) throws
     String lvStr = getProperty(props, OzoneConsts.LAYOUTVERSION, versionFile);
 
     int lv = Integer.parseInt(lvStr);
-    if(DataNodeLayoutVersion.getLatestVersion().getVersion() != lv) {
+    boolean validVersion = Stream.of(DataNodeLayoutVersion.getAllVersions())
+        .anyMatch(dataNodeLayoutVersion -> (
+            dataNodeLayoutVersion.getVersion() == lv));
+    if (!validVersion) {
+      int[] supportedVs =
+          Stream.of(DataNodeLayoutVersion.getAllVersions())
+              .mapToInt(DataNodeLayoutVersion::getVersion).toArray();
       throw new InconsistentStorageStateException("Invalid layOutVersion. " +
-          "Version file has layOutVersion as " + lv + " and latest Datanode " +
-          "layOutVersion is " +
-          DataNodeLayoutVersion.getLatestVersion().getVersion());
+          "Version file has layOutVersion as " + lv + " which is unsupported" +
```

Comment:
Can we change layOut to layout. Made this mistake previously but since layout is single word, it would be good to remove the capital O.

Reply:
Done.